### PR TITLE
MIT LICENSE

### PR DESCRIPTION
**Pull request by AknEp**
#21

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017 AknEp
```
Comment (englercj) [inline LICENSE.txt]: This repository should include the copyright notice from Pixi: https://github.com/pixijs/pixi.js/blob/dev/LICENSE

It should be copyright by myself, Mat, or both of us.

File: LICENSE.txt [hunk 2]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (englercj) [inline LICENSE.txt]: This should be "The MIT License", I recommend copy-pasting the license from pixi itself: https://github.com/pixijs/pixi.js/blob/dev/LICENSE

File: LICENSE.txt (hunk 1 above)
Comment (AknEp) [inline LICENSE.txt]: ops, sorry I'll change it.

**Comment by AknEp** [comment]
updated. please check it.

**Comment by AknEp** [comment]
Thank you! 👍

**Pull request merged**